src/open_inwoner/configurations/management/commands/send_failed_mail_digest.py

```diff
@@ -0,0 +1,21 @@
+import logging
+
+from django.core.management.base import BaseCommand
+
+from open_inwoner.configurations.emails import inform_admins_about_failing_emails
+
+logger = logging.getLogger(__name__)
```

Comment:
You're defining a logger but then only write to `stdout`. You can either get rid of the logger or use it in the `handle`.